explanation/template-explanation.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Explanation template
 // Article variables (delete the comments and add in the values)
-:description: Summarise what this explanation is about. What you put here is reused in the Overview section and included in HTML description tags.
+:description: Summarize what this explanation is about. What you put here is reused in the Overview section and included in HTML description tags.
```

Comment:
This is correct. 

For wider discussion (and non-blocking for this PR), is it time to move these templates into MD rather than AsciiDoc to make them consistent with the other templates offered?

Reply:
Ugh, I thought I backed out my changes to this file, but it's no big deal. See my 'notes for reviewers' in the PR description.